README.md

```diff
@@ -150,7 +150,12 @@ are loaded for the agent at any given time.
 3. Find your API key and region under the default project in the left sidebar.
 
 ### Setting up environment variables
-   For Windows Users:
+
+Simply set them in the `.env` file. 
+
+Alternatively you can use the appropriate following method (advanced):
```

Comment:
```suggestion
Alternatively you can use set them from the command line (advanced):
```

Reply:
Thanks, that looks better. Can I approve of this some way? Do I need to do something? Sorry not that well versed with GitHub.